pkg/cli/find/find.go

```diff
@@ -281,7 +281,10 @@ func ListVClusters(ctx context.Context, context, name, namespace string, log log
 	}
 	// Find virtual cluster instances, so we can pair them with OSS virtual clusters.
 	virtualClusterInstancesList, err := kubeClient.Loft().StorageV1().VirtualClusterInstances("").List(ctx, listOptions)
-	if err != nil {
+	if kerrors.IsForbidden(err) {
```

Comment:
Pairing VirtualClusterInstances to found OSS virtual clusters is done on a best effort basis, as VirtualClusterInstance CRD maybe does not even exist on the cluster (e.g. in case of OSS vcluster, or connected host cluster).

So here we have another scenario where this pairing will not work - when the user that is running the CLI does not have permissions to list VirtualClusterInstances. In this case the func just return the OSS virtual clusters that it found so far (like few lines above where it cannot find VirtualClusterInstance CRD).